dao/src/main/resources/db/changelog/logs/ch-add-trigger_for_calculate_sum_total_amount_without_discounts_bag_table-Kyzbyt_Maksym.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="Kuzbyt-4" author="Maksym Kuzbyt">
+        <sql endDelimiter="/">
+
+            CREATE TRIGGER insert_update_bag
```

Comment:
please check if you can use generated columns instead https://www.postgresql.org/docs/12/ddl-generated-columns.html